src/api/app/controllers/webui/project_controller.rb

```diff
@@ -336,7 +336,7 @@ def clear_failed_comment
     flash.now[:success] = 'Cleared comments for packages'
```

Comment:
clear failed comment is about the same view, so it would make sense to move as well

Reply:
yes, however not in this PR. But it's in my TODO list too, thanks @coolo